cdc/processor/pipeline/sink.go

```diff
@@ -286,32 +283,17 @@ func splitUpdateEvent(updateEvent *model.PolymorphicEvent) (*model.PolymorphicEv
 	return &deleteEvent, &insertEvent, nil
 }
 
-// clearBuffers clears rowBuffer.
-// Also, it dereferences data that are held by buffers.
-func (n *sinkNode) clearBuffers() {
-	// Do not hog memory.
-	if cap(n.rowBuffer) > defaultSyncResolvedBatch {
-		n.rowBuffer = make([]*model.RowChangedEvent, 0, defaultSyncResolvedBatch)
-	} else {
-		for i := range n.rowBuffer {
-			n.rowBuffer[i] = nil
-		}
-		n.rowBuffer = n.rowBuffer[:0]
-	}
-}
-
 // emitRowToSink emits the rows in rowBuffer to backend sink.
-func (n *sinkNode) emitRowToSink(ctx context.Context) error {
+func (n *sinkNode) emitRowToSink(ctx context.Context, rows ...*model.RowChangedEvent) error {
```

Comment:
Perhaps we can move this failpoint to tablesink to simplify the logic here. For example, remove emitRowToSink and directly emit RowChangedEvents to table sink.

Reply:
I'll trt it.

Reply:
Move it into new emitRowToSink func.